installer/functions/plugin/helper-plugins.private.ts

```diff
@@ -0,0 +1,163 @@
+
+import axios from 'axios';
+import { ApplicationContext, Configuration } from '../../src/constants/interface';
+import { 
+  ACCOUNT_SID,
+  AUTH_TOKEN,
+  getParam
+} 
+from '../common';
+
+export const listPluginsEndpoint = "https://flex-api.twilio.com/v1/PluginService/Plugins";
+
+/**
+ * Needed to make API calls
+ * @param stringToConvert 
+ * @returns - base64 string
+ */
+export const convertToBase64 = (stringToConvert: string) => {
+  return Buffer.from(stringToConvert).toString('base64');
+}
+
+/**
+ * 
+ * @param accountSid 
+ * @param authToken 
+ * @returns - base64 User Credentials
+ */
```

Comment:
just a personal preference, but JSDoc strings aren't necessary for TS since types are themselves self-documenting; just makes code a bit more unnecessarily verbose. but by all means if you need to explain the "why" of a piece of code, comments are totally valid